pages/index.js

```diff
@@ -139,253 +124,336 @@ class Index extends Component {
-                    })}
+                    </StaggeredMotion>
+                  }
+                </Delay>
+                <section
+                  className="logo-collection"
+                  ref={(logos) => { this.logos = logos }}
+                  >
+                  <h3
+                    style={{
+                      margin: 0,
+                    }}
                     >
-                    {interpolatingStyles =>
-                      <ul>
-                        {interpolatingStyles.map((style, i) => (
-                          <BookLI
-                            key={i} // eslint-disable-line
-                            style={{
-                              transform: `translateX(-${style.left}px)`,
-                              opacity: style.opacity,
-                            }}
-                            >
-                            {bookTopics[i]}
-                          </BookLI>
-                        ))}
-                      </ul>
-                    }
-                  </StaggeredMotion>
-                }
-              </Delay>
-              <div
-                style={{
-                  display: 'flex',
-                  width: 450,
-                  justifyContent: 'space-between',
-                  marginLeft: 30,
-                  opacity: 0,
-                }}
-                className="logo-collection"
-                ref={(logos) => { this.logos = logos }}
-                >
-                { // eslint-disable-next-line
-                }<svg xmlns="http://www.w3.org/2000/svg" width="600" height="600" viewBox="0 0 600 600"><circle fill="#00D8FF" cx="299.529" cy="299.628" r="50.167"/><path fill="none" stroke="#00D8FF" strokeWidth="24" strokeMiterlimit="10" d="M299.53 197.628c67.355 0 129.927 9.665 177.106 25.907 56.844 19.57 91.794 49.233 91.794 76.093 0 27.99-37.04 59.503-98.083 79.728-46.15 15.29-106.88 23.272-170.818 23.272-65.555 0-127.63-7.492-174.29-23.44-59.047-20.183-94.612-52.104-94.612-79.56 0-26.642 33.37-56.076 89.415-75.616 47.355-16.51 111.472-26.384 179.486-26.384z"/><path fill="none" stroke="#00D8FF" strokeWidth="24" strokeMiterlimit="10" d="M210.736 248.922c33.65-58.348 73.28-107.724 110.92-140.48 45.35-39.466 88.507-54.923 111.775-41.505 24.25 13.983 33.043 61.814 20.068 124.796-9.81 47.618-33.234 104.212-65.176 159.6-32.75 56.79-70.25 106.82-107.377 139.273-46.98 41.068-92.4 55.93-116.185 42.213-23.08-13.31-31.906-56.922-20.834-115.234 9.355-49.27 32.832-109.745 66.81-168.664z"/><path fill="none" stroke="#00D8FF" strokeWidth="24" strokeMiterlimit="10" d="M210.82 351.482c-33.745-58.292-56.73-117.287-66.31-166.255-11.545-59-3.383-104.11 19.863-117.566 24.224-14.023 70.055 2.245 118.14 44.94 36.356 32.28 73.688 80.838 105.723 136.174 32.844 56.733 57.46 114.21 67.036 162.582 12.117 61.213 2.31 107.984-21.453 121.74-23.058 13.348-65.25-.784-110.24-39.5-38.013-32.71-78.682-83.252-112.76-142.114z"/></svg>
-                { // eslint-disable-next-line
```

Comment:
either keep these in to prevent eslint error, or change the svgs to be compliant (like the bottom svg)